src/Lib/Util/FileSystem.cpp

```diff
@@ -29,12 +30,14 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 namespace filesystem {
 
-void initialize(const char* argv0, const char*, const char* application) {
```

Comment:
This sure looks funny to me - a type name but no arg after it.

Reply:
Coding with notepad.exe :)